Tests/scripts/download_conf_repos.sh

```diff
@@ -85,7 +85,17 @@ clone_repository_with_fallback_branch() {
   fi
 }
 
-echo "Getting content-test-conf and infra repositories with branch:${CI_COMMIT_BRANCH}, with fallback to master"
+TEST_UPLOAD_BRANCH_SUFFIX="-upload_test_branch-"
```

Comment:
test-upload-flow sub-pipeline didn't know how to find the matching branch in infra and content-test-conf until now, added this feature.